ui/src/views/infra/network/providers/AddF5LoadBalancer.vue

```diff
@@ -242,17 +242,9 @@ export default {
           }
           params.id = this.nsp.id
           const jobId = await this.addF5LoadBalancer(params)
-          if (jobId) {
-            await this.$store.dispatch('AddAsyncJob', {
-              title: this.$t(this.action.label),
-              jobid: jobId,
-              description: this.$t(this.nsp.name),
-              status: 'progress'
-            })
-            await this.parentPollActionCompletion(jobId, this.action)
-          }
-          this.loading = false
+          await this.parentPollActionCompletion(jobId, this.action, this.$t(this.nsp.name))
```

Comment:
Don't need to wait for the job to be completed, can close the form once submitted